test/integration/targets/elb_network_lb/tasks/test_creating_nlb.yml

```diff
@@ -27,13 +27,23 @@
           DefaultActions:
             - Type: forward
               TargetGroupName: "{{ tg_name }}"
+        - Protocol: UDP
+          Port: 13
+          DefaultActions:
+            - Type: forward
+              TargetGroupName: "{{ tg_name }}"
+        - Protocol: TCP_UDP
+          Port: 17
+          DefaultActions:
+            - Type: forward
+              TargetGroupName: "{{ tg_name }}"
```

Comment:
The target group created in `test/integration/targets/elb_network_lb/tasks/main.yml` is TCP-only, and the TLS listener in this task can not go to a TCP_UDP target group.  You'll need to create a second TCP_UDP target group in `test/integration/targets/elb_network_lb/tasks/main.yml` and a second NLB to test the new protocols, then be sure all the additional resources are cleaned up in their respective task file's `always` block.

Reply:
thank you for point me to right path! :D